src/Microsoft.SqlTools.ServiceLayer/EditData/Contracts/EditCell.cs

```diff
@@ -0,0 +1,35 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.EditData.Contracts
+{
+    /// <summary>
+    /// Cell that wraps info from <see cref="DbCellValue"/> for edit purposes
+    /// </summary>
+    public class EditCell : DbCellValue
+    {
+        /// <summary>
+        /// Constructs a new EditCell based on a DbCellValue
+        /// </summary>
+        /// <param name="dbCellValue">The DbCellValue that will be enhanced</param>
+        /// <param name="isDirty">Whether or not the edit cell is dirty</param>
+        public EditCell(DbCellValue dbCellValue, bool isDirty)
```

Comment:
I feel like it's super weird to pass the parent class into the constructor of the child class.